message_definitions/v1.0/messages.xml

```diff
@@ -2480,6 +2480,19 @@
       <field name="course"  type="float" unit="deg"/>
     </message>
 
+    <message name="TARGET_POS" id="57" link="forwarded">
+      <description>
+        Ground station GPS position for functions like Follow Me
```

Comment:
The description should be changed as well as it is not limiter to ground station